docs/index.md

```diff
@@ -20,7 +21,7 @@ Next, run an LLM:
     lamini.api_key = "<YOUR-LAMINI-API-KEY>"
 
     llm = lamini.Lamini("meta-llama/Meta-Llama-3-8B-Instruct")
-    print(llm.generate("How are you?"))
+    print(llm.generate("How are you?", output_type={"Response":"str"}))
```

Comment:
seems like i added this back because the response in the docs says "Response", of course we may not need this but it should be consistent

Reply:
Additionally, if we change this we should also change the curl version

Reply:
I vote that we use output_type across the board in our docs. I'll take inventory.

The upside: structured output is very useful and a big differentiator for us, so we should train people to use it everywhere.

The downside: it's one more moving piece that someone could get wrong while they're learning the Lamini basics, especially if they're hacking around and editing examples (if we have an example with `int` output and they edit the prompt to ask for text, they're gonna have a confusing time).